### DISCUSSION TITLE
YouTube Netflix 视频播放上一句, 重复这句, 下一句功能.

### DISCUSSION THREAD

**Discussion by thomasTAT** [immersive-translate/immersive-translate#1892] [Ideas]
### 新功能 | New Feature

YouTube Netflix 视频播放上一句, 重复这句, 下一句功能.
类似 [language reactor](https://www.languagereactor.com/m/um_en_-) 和 [Language Learning with Netflix & YouTube-AFL](https://chromewebstore.google.com/detail/bekopgepchoeepdmokgkpkfhegkeohbl) 用 A S W 去控制.

### 为什么您想要添加这些功能 | Why you want to add these features

方便学习英语.

**Comment by theowenyoung** [comment]
您好，目前无计划添加更多英语学习相关的功能，建议您直接使用 Language Reactor 即可。
